Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -57,7 +57,7 @@ open class ChatMessageContentView: _View, ThemeProvider {
     }
 
     /// The date formatter of the `timestampLabel`
-    public lazy var dateFormatter: DateFormatter = .makeDefault()
+    public lazy var dateFormatter: DateFormatter = .messageTimestamp
```

Comment:
doc here does not make a lot of sense

Reply:
I've updated them a little bit now, but it actually makes sense, this is the formatted used by the `timestampLabel` in this case. Were you thinking of a better doc?